Yoast/wordpress-seo
# Fixed the situation where options were being changed when toggling features

## Summary

This PR can be summarized in the following changelog entry:

* Fixed a bug where certain options (`site_type`, `environment_type` and `has_multiple_authors`) would be reset to their default value whenever one of the feature toggles were changed.
* Ensured that `has_multiple_authors` gets validated.

## Test instructions

This PR can be tested by following these steps:

* Go through the steps and complete the Configuration Wizard.
* Take note of the values that were set for the Site Type, Environment Type and Has Multiple Authors option.
* Change any of the feature toggles to a different value and save it.
* Ensure that the three aforementioned settings remain unchanged after enabling/disabling a particular feature.

Fixes #6021
